_data/sites.json

```diff
@@ -12300,6 +12300,13 @@
         ]
     },
 
+    {
+        "name": "TomTom GO",
+        "url": "https://help.tomtom.com/hc/en-gb/requests/new?ticket_form_id=360000697820",
+        "difficulty": "medium",
+        "notes": "Fill in the form to automatically open a support ticket."
```

Comment:
If it requires support to be involved, and thus manual human labour, we classify it as hard